### PR TITLE
Changes to PagerDuty README

### PR DESCRIPTION
**Pull request by jfryman**
Few minor changes to the PD Readme.

### CODE REVIEW

File: packs/pagerduty/README.md [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
-3. Close and open incident or you can even close multiple open incidents by providing comma sperated values
+2. Send acknowledgment of any incident(s)
+3. Close and open incident(s)
 4. Launch an incident by giving its details and description
 
-# Confgiuration
+# Configuration
 
-You need the following parameters to be in the config.yaml file
 `subdomain: `'name of subdomain '
```
Comment (Kami) [inline packs/pagerduty/README.md]: It would be nice if this used bullet points so it's consistent with other readmes.

Comment (jfryman) [inline packs/pagerduty/README.md]: Do we have a canonical README that we can agree on? We're all over the place on this point...

Comment (Kami) [inline packs/pagerduty/README.md]: Most of the readmes I've touched (that's probably at least 60% of the packs) follow the "canonical" format. Here is an example - https://github.com/StackStorm/st2contrib/blob/master/packs/slack/README.md

I also want to get this merged - https://github.com/StackStorm/st2contrib/pull/87, but first I need to update a couple of readmes which are left which still don't conform to this format.

Comment (jfryman) [inline packs/pagerduty/README.md]: Yeah, #87 would be stellar to consistency. :+1:

**Pull request merged**